### PR TITLE
Final css fixes

### PR DESCRIPTION
**Pull request by awongh**

### CODE REVIEW

File: _posts/2021-08-11-do-i-need-a-computer-science-degree-to-be-a-software-engineer.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: Do I Need a Computer Science Degree to Be a Software Engineer?
 meta_description: ../../assets/img/blog/blog-cs-degree.jpg
 author:
 date: "2021-08-11"
-categories: "learning methods"
+categories: ["learning", "methods"]
```
Comment (awongh) [inline _posts/2021-08-11-do-i-need-a-computer-science-degree-to-be-a-software-engineer.md:7]: I think this should be in a different commit, bc this one is for br tags

File: _posts/2021-08-12-postgraduate-software-engineering-programs-in-singapore.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ title: Postgraduate Software Engineering Programmes in Singapore
 meta_description: ../../assets/img/blog/blog-postgrad.jpg
 author:
 date: "2021-08-12"
-categories: learning methods
```
Comment (awongh) [inline _posts/2021-08-12-postgraduate-software-engineering-programs-in-singapore.md:7]: is the correction for th elarning methods above the same as this one?

I would have assumed it was ok to put the entire thing into a string, like the original above?

this change strongly imples that there are two categories, "learning" and "methods" - but is it acutally a single category?

File: _sass/partials/_about.scss [hunk 3]
```diff
@@ -1,5 +1,4 @@
 .about-header-row {
-  height:30vh;
```
Comment (awongh) [inline _sass/partials/_about.scss]: I added this CSS here: https://github.com/rocketacademy/rocketacademy.co/pull/9/commits/d2699024505d07e271acae78583b2bde4f12fd95#diff-4a4bfc24b8a25a96f0ebb3df457b0e123c84317b82b5bbf75db81db09ece7400R2

was taking it out intentional?

File: _sass/partials/_navbar.scss [hunk 4]
```diff
@@ -2,7 +2,6 @@
   height: 69px;
   background-color: $light;
   letter-spacing: 0.1rem;
-  padding: 0 10px 0 0;
```
Comment (awongh) [inline _sass/partials/_navbar.scss]: same here: https://github.com/rocketacademy/rocketacademy.co/pull/9/commits/cf7a787e91f493c52f4e576c59cb3e6e133ed2ac#diff-3bbfc66923ee189393dadae90253c2476a40a5a5a75742aa7ad715e4755bb166R5

was this taken out intentionally?

File: _sass/partials/_about.scss (hunk 3 above)
Comment (michelle-mok) [reply _sass/partials/_about.scss]: yes, couldn't see the text below the header in mobile view

File: _sass/partials/_navbar.scss (hunk 4 above)
Comment (michelle-mok) [reply _sass/partials/_navbar.scss]: yes, there was a gap between the navbar dropdown and edge of page in mobile/tablet view

File: _posts/2021-08-12-postgraduate-software-engineering-programs-in-singapore.md (hunk 2 above)
Comment (michelle-mok) [reply _posts/2021-08-12-postgraduate-software-engineering-programs-in-singapore.md:7]: it is actually a single category but jekyll reads it as an array, i couldn't get the space between the 2 words, "learning methods" still ends up "learningmethods"

File: _sass/partials/_navbar.scss (hunk 4 above)
Comment (awongh) [reply _sass/partials/_navbar.scss]: ah good catch! If you find something like that I think it would be easier to just comment and let me know.

my original code was fixing a different spacing problem

File: _posts/2021-08-12-postgraduate-software-engineering-programs-in-singapore.md (hunk 2 above)
Comment (awongh) [reply _posts/2021-08-12-postgraduate-software-engineering-programs-in-singapore.md:7]: ok, I see
